base/int.jl

```diff
@@ -860,3 +860,67 @@ for op in (:+, :-, :*, :&, :|, :xor)
         return $op(aT, bT)
     end
 end
+
+"""
+    bitreverse(x)
+
+Reverse the order of bits in integer `x`. `x` must have a fixed bit width,
+e.g. be an `Int16` or `Int32`.
+
+!!! compat "Julia 1.5"
+    This function requires Julia 1.5 or later.
+
+# Examples
+```jldoctest
+julia> bitreverse(0x8080808080808080)
+0x0101010101010101
+
+julia> reverse(bitstring(0xa06e)) == bitstring(bitreverse(0xa06e))
+true
+```
+"""
+function bitreverse end
+
+function bitreverse(x::Union{Int8,UInt8})
+    z = x % UInt8
+    z = ((z >>> 1) & 0x55) | ((z << 1) & 0xaa)
```

Comment:
why the mix of arithmetic and logic shifts?

Reply:
There is no `<<<`?